src/ImageProcessor/Colors/ColorTransforms.cs

```diff
@@ -67,6 +67,31 @@ public partial struct Color
             return new Color(r, g, b);
         }
 
+        public static implicit operator Color(CieXyz color)
+        {
+            var x = color.X;
+            var y = color.Y;
+            var z = color.Z;
+
+            // assume sRGB
+            // http://www.brucelindbloom.com/index.html?Eqn_RGB_XYZ_Matrix.html
+            float r = (x * 3.240969941904521F) + (y * -1.537383177570093F) + (z * -0.498610760293F);
+            float g = (x * -0.96924363628087F) + (y * 1.87596750150772F) + (z * 0.041555057407175F);
+            float b = (x * 0.055630079696993F) + (y * -0.20397695888897F) + (z * 1.056971514242878F);
+
+            r = r > 0.0031308 ? (float)((1.055 * Math.Pow(r, 1.0 / 2.4)) - 0.055) : r = (r * 12.92F);
+
+            g = g > 0.0031308 ? (float)((1.055 * Math.Pow(g, 1.0 / 2.4)) - 0.055) : g = (g * 12.92F);
+
+            b = b > 0.0031308 ? (float)((1.055 * Math.Pow(b, 1.0 / 2.4)) - 0.055) : b = (b * 12.92F);
+
+            r = Math.Min(Math.Max(0, r), 1);
```

Comment:
Got you covered already. 
Use `Color.Compand(color)`
Return `color.Limited`. :smile: